torch/_tensor.py

```diff
@@ -1011,6 +1011,33 @@ def rename(self, *names, **rename_map):
         # See Note [rename_ / rename API]
         return update_names(self, names, rename_map, inplace=False)
 
+    def to_sparse_coo(self):
+        """ Convert a tensor to coordinate format. Only works with 2D tensors.
+
+       Examples::
+
+            >>> dense = torch.randn(5, 5)
+            >>> sparse = dense.to_sparse_coo()
+            >>> sparse._nnz()
+            25
+
+       """
+        if self.is_sparse and not self.is_sparse_csr:
```

Comment:
`Tensor.is_sparse` means "is sparse COO tensor", hence:
```suggestion
        if self.is_sparse:
```